Misc/NEWS.d/next/Library/2020-03-21-00-46-18.bpo-40017.HFpHZS.rst

```diff
@@ -0,0 +1 @@
+Expose the CLOCK_TAI constant to the time library if the operating system support it.
```

Comment:
```suggestion
Add :data:`time.CLOCK_TAI` constant, if the operating system support it: International Atomic Time.
```

Reply:
You and @benjaminp seem to have different ideas for this entry. I've gone with @vstinner's version for now, but @benjaminp if you want something else perhaps the two of you can agree?